spec/components/previews/provider_interface/change_offer_preview.rb

```diff
@@ -0,0 +1,151 @@
+module ProviderInterface
+  class ChangeOfferPreview < ViewComponent::Preview
+    def existing_offer_start_at_1_provider(provider_interface_change_offer_form: nil)
+      initial_step :provider
+      set_application_choice application_choice_with_offer
+
+      form = submitted_form(provider_interface_change_offer_form) || new_form
+      render_with form
+    end
+
+    def existing_offer_start_at_2_course(provider_interface_change_offer_form: nil)
+      initial_step :course
+      set_application_choice application_choice_with_offer
+
+      form = submitted_form(provider_interface_change_offer_form) || new_form
+      render_with form
+    end
+
+    def existing_offer_start_at_3_location(provider_interface_change_offer_form: nil)
+      initial_step :course_option
+      set_application_choice application_choice_with_offer
+
+      form = submitted_form(provider_interface_change_offer_form) || new_form
+      render_with form
+    end
+
+    def existing_offer_start_at_confirm_change(provider_interface_change_offer_form: nil)
+      initial_step :confirm
+      set_application_choice application_choice_with_offer
+
+      form = submitted_form(provider_interface_change_offer_form)
+      form ||= new_form(new_course_option: pick_different_option)
+      render_with form
+    end
+
+    def new_offer_start_at_1_provider(provider_interface_change_offer_form: nil)
```

Comment:
Similar repetition here suggests these 3 methods could be simplified and DRY'd up.